ibm/config.go

```diff
@@ -2738,6 +2752,39 @@ func (c *Config) ClientSession() (interface{}, error) {
 			"X-Original-User-Agent": {fmt.Sprintf("terraform-provider-ibm/%s", version.Version)},
 		})
 	}
+	
+	//COMPLIANCE Service v2 version
+	// Construct an "options" struct for creating the service client.
+	var postureManagementClientURLv2 string
+	if c.Visibility == "public" || c.Visibility == "public-and-private" {
+		postureManagementClientURLv2, err = posturemanagementv2.GetServiceURLForRegion(c.Region)
+	} else {
+		session.findingsClientErr = fmt.Errorf("Error occurred while configuring Security Insights Findings API service: `%v` visibility not supported", c.Visibility)
```

Comment:
should be `postureManagementClientErrv2` not `findingsClientErr`


could you please change this in v1 client initialisation as well..
It has to be `postureManagementClientErr` [here](https://github.com/IBM-Cloud/terraform-provider-ibm/blob/a0d7985697370c22c1991ae74d4ad8e9c66517fa/ibm/config.go#L2728)